src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem/TodoItem';
+
+export type Props = {
+  todos: Todo[],
+  onDeleteTodo(id: number): void,
+};
+
+export const TodoList: React.FC<Props> = ({ todos, onDeleteTodo }) => {
+  return (
+    <ul className="todoapp__main" data-cy="TodoList">
+      {todos.map(todo => (
+        <TodoItem
+          key={todo.id}
+          todo={todo}
+          onDeleteTodo={onDeleteTodo}
+        />
```

Comment:
Semantically better to extract `li` to the same component with `ul`
```suggestion
        <li ...>
          <TodoItem
            key={todo.id}
            todo={todo}
            onDeleteTodo={onDeleteTodo}
          />
        </li>
```